tpdatasrc/tpgamefiles/rules/d20_combat/damage_critter.py

```diff
@@ -2,10 +2,88 @@
 import roll_history
 import logbook
 
+def playSoundEffect(weaponUsed, attacker, tgt, sound_number):
+    missing_stub("auto soundId = inventory.GetSoundIdForItemEvent(weaponUsed, attacker, tgt, 6); sound tbd")
+    missing_stub("sound.PlaySoundAtObj(soundId, attacker); sound tbd")
+    return
+
 def deal_attack_damage(attacker, tgt, d20_data, flags, action_type): 
     # return value is used by Coup De Grace
     print('Quel dommage')
-    #...
+
+    #Get Python damage
+    missing_stub("auto pyResult = DealAttackDamagePython(attacker, tgt, d20Data, flags, actionType)")
+
+    #Provoke Hostility
+    missing_stub("aiSys.ProvokeHostility(attacker, tgt, 1, 0);")
+
+    #Check if target is alive
+    missing_stub("if critterSys.IsDeadNullDestroyed(tgt), is it enough to just query if dead?")
+    if tgt.d20_query(Q_Dead):
```

Comment:
I don't think this is equivalent. Stop using this please, you did it in the other function too and it was a bug.